### DISCUSSION TITLE
Weird address location behavior

### DISCUSSION THREAD

**Discussion by meyverick** [osmandapp/OsmAnd#17058] [Q&A]
I updated the following house on Osm: 50.02232 4.15628 .

The address is "Rue de Beauwelz 40, 6590 Momignies". As you'll see, I added an Entrance point because the entrance is from the rear of the building, on "Le Trieu Bayard" street.

I updated OsmAND maps and the address is visible on the map but I can't reach it using the address search.

Can you explain what's wrong?

**Comment by tommycrock** [comment]
I think addresses don't become searchable until a monthly update is done. That seems to have been my experience anyway. Not sure if it's a known issue with live updates.
